lisa/wa_results_collector.py

```diff
@@ -540,41 +582,12 @@ def _get_extra_job_metrics(self, job_dir, workload):
                 continue
 
             if artifact_name.startswith('energy_instrument_output'):
-
-                try:
-                    df = pd.read_csv(path)
-                except pandas.errors.ParserError as e:
+                df = self._read_energy_instrument_metrics(path)
```

Comment:
following previous suggested change, this would turn into:
```
try:
    df = self._read_energy_instrument_metrics(path) 
except pandas.errors.ParserError as e:
     logger.info(" no data for %s", path)
     continue
else:
      .... # do something with df
```

Reply:
Done